dockerfile

```diff
@@ -1,69 +1,55 @@
-FROM nikolaik/python-nodejs:python3.8-nodejs14@sha256:f7d84950cac6a56e99e99747aad93ad77f949475ac1a5e9782669b8f5bedd2b6 AS base
-
-WORKDIR /usr/app
-
-
 # -----------------------------
-# Stage 1: install dependencies
+# Stage 0: install dependencies
 # -----------------------------
-FROM base AS dependencies-builder
-
-# odbc - databricks
-RUN apt-get update && apt-get install -y --no-install-recommends \
-    wget \
-    libsasl2-modules-gssapi-mit
-RUN wget \
-    --quiet \
-    https://databricks-bi-artifacts.s3.us-east-2.amazonaws.com/simbaspark-drivers/odbc/2.6.18/SimbaSparkODBC-2.6.18.1030-Debian-64bit.zip \
-    -O /tmp/databricks_odbc.zip \
-    && unzip /tmp/databricks_odbc.zip -d /tmp \
-    && dpkg -i /tmp/simbaspark_*.deb
-
-
-# -------------------------------
-# Stage 2: base with dependencies
-# -------------------------------
-FROM base as base-dependencies
+FROM node:14-bullseye AS base
+WORKDIR /usr/app
 
-# odbc
-COPY --from=dependencies-builder /opt/simba /opt/simba
-# TODO: prefer in stage 1 - needed for yarn install and pip install
 RUN apt-get update && apt-get install -y --no-install-recommends \
+    build-essential \
     g++ \
+    libsasl2-modules-gssapi-mit \
+    nodejs \
+    python3 \
+    python3-psycopg2 \
+    python3-venv \
+    software-properties-common \
     unixodbc-dev \
-    software-properties-common
-
-# Install latest git
```

Comment:
@jim80net I merged this yesterday but dropped `git` from the container, do we need to add this to the `apt` package list?

Reply:
For git, no. git is available by default in node:14-bullseye

![image](https://user-images.githubusercontent.com/176915/154144237-82fbe83c-21fe-45ee-b494-64348a3e4997.png)

Reply:
That's so bizarre, I'll test out now

Reply:
I think I found what was missing: 65dd8fd: I was using `node:14-slim` when I thought it was `node:14-bullseye`.